share/spice/syntax/syntax.js

```diff
@@ -0,0 +1,32 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_syntax = function(api_result) {
+        if (!api_result || api_result.error || api_result.length == 0) {
+            return Spice.failed('syntax');
+        }
+        var query = DDG.get_query().replace("syntax", '');
+        Spice.add({
+            id: "syntax",
```

Comment:
@anthonyn60 Heere!

@zekiel Might we not also have to update the share directory?